content/klc/S2.2.adoc

```diff
@@ -4,16 +4,12 @@ brief = "Non-functional variations in part number should be replaced with wildca
 
 Manufacturer Part Numbers (MPN) often include variations that are _non-functional_.
 
-Examples:
+Examples of parameters considered non-functional:
 
 * Temperature rating of the component
 * Packaging information (e.g. reel, tray, tape)
 * RoHS / PbFree information
 
-To capture every possible combination of part variation requires a large number of symbol aliases - this is to be avoided.
+To capture every possible combination of part variation would require a large number of symbol aliases - this is to be avoided.
 
-Where an MPN has options for such non-functional variations, these portions of the MPN should be replaced with a wildcard character (`x`).
-
-**Rule of thumb**
-
-A single symbol should be drawn for each footprint variation of an atomic part. Other variations (as listed above) do not require their own symbol alias.
+Where an MPN has options for such non-functional variations, these portions of the MPN should be replaced with a wildcard character (`x`). If these non-functionl parts of the MPN are at the end of it then these have to be omitted.
```

Comment:
'at the end of it' isn't clear; what is being referred to because it's not the MPN which gets a wildcard. Change 'it then these have to' to 'the symbol name then the wildcard should'? Also there is a missing 'a' in the second instance of 'functional'.

Reply:
I am not sure your description makes more sense to be honest. Maybe i misunderstood your partial inserts.

Reply:
Yes. That could be. Let me share the entire sentence I suggest to replace the final one above: `If these non-functional parts of the MPN are at the end of the symbol name then the wildcard should be omitted.`.

Either way, there is an 'a' missing in 'functional'.

Reply:
done